cmd/trade.go

```diff
@@ -81,9 +85,9 @@ func init() {
 
 		if *fixedIterations == 0 {
 			fixedIterations = nil
-			log.Printf("will run unbounded iterations\n")
+			l.Info("will run unbounded iterations")
```

Comment:
validateCliParams() can take in the a `l logger.Logger` as a param so it can be used here